CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### CHANGED
+
+- Upgraded for usage with terraform-0.12.x [#6]
```

Comment:
The URL for the #6 link must be added to the bottom of the file.

Reply:
I'll add it now

Reply:
This is also wrong issue number. change it to #5